AlexBeerShop_MVVM/Views/Controllers/RandomBeerVC.swift

```diff
@@ -10,4 +10,77 @@ import UIKit
 
 final class RandomBeerVC: UIViewController {
     
+    //MARK: - Elements
+//    let detailVC =  DetailVC(beer: <#SingleBeer#>)
+    let randomBeerViewModel  = RandomBeerViewModel()
+    private let beerViewTamplate = DetailBeerView()
+    private let randomButton: UIButton = {
+        let randomButton = UIButton()
+        randomButton.backgroundColor = .orange
+        randomButton.setTitle("Random beer search", for: .normal)
+        randomButton.isEnabled = true
+        return randomButton
+    }()
+    
+    
+    //MARK: - Lifecycle
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setupVC()
+        setupUI()
+        addTargets()
+    }
+    init() {
```

Comment:
For what there r initializator?

Reply:
'required' initializer 'init(coder:)' must be provided by subclass of 'UIViewController'